docs/usage.rst

```diff
@@ -135,15 +129,10 @@ The full list of arguments can be accessed via :bash:`pyreisejl/utility/call.py
                             will be deleted. The extraction process can be memory
                             intensive. This is optional and defaults to False if the
                             flag is omitted.
-      -o OUTPUT_DIR, --output-dir OUTPUT_DIR
+      -o [OUTPUT_DIR], --output-dir [OUTPUT_DIR]
                             The directory to store the extracted data. This is optional
                             and defaults to the execute directory. This flag is only
```

Comment:
`execute directory` --> `input_directory`

Reply:
updated - default is a folder in the input directory